Sources/swift-doc/Supporting Types/Page.swift

```diff
@@ -43,3 +41,11 @@ func path(for symbol: Symbol) -> String {
 func path(for identifier: CustomStringConvertible) -> String {
     return "\(identifier)".replacingOccurrences(of: ".", with: "_")
 }
+
+func writeFile(_ data: Data, to url: URL) throws {
```

Comment:
Perhaps `writeFile(with data...` would have been better. Or `write(_ data` but that one gave me some errors I wasn't able to track.